404.html

```diff
@@ -15,8 +15,9 @@ <h1 class="title">
 
       <nav class="level">
           <div class="level-item has-text-centered">
-              <a class="button is-danger icon-button">Homepage</a>
-              <a class="button is-danger">Documentation</a>
+              <a class="button is-danger icon-button"> Homepage </a>
+              <a class="button is-danger icon-button"> Documentation </a>
+              <a class="button is-danger"> Releases </a>
```

Comment:
Releases shouldn't be shown on a 404 page.

Reply:
Complete.